src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivityMenu.test.js

```diff
@@ -15,16 +23,16 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivityMe
     `(
         `for an activity with permissions $permissions and onEdit ($onEdit), should showDelete: $showDelete, showEdit: $showEdit`,
         ({ permissions, showDelete }) => {
-            const wrapper = getWrapper({ ...permissions });
+            const wrapper = getWrapper({ ...permissions, features: defaultFeatures });
```

Comment:
Could we move `features: defaultFeatures` into a `default` props object that gets spread in the `getWrapper` method? 
```
const getWrapper = (props = {}) => shallow(<AnnotationActivityMenu id="123" {...defaults} {...props} />);
```